pkg/cmd/status/status.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 const (
-	kafkaSvcName = "kafka"
+	kafkaSvcName    = "kafka"
+	registrySvcName = "registry"
 )
 
-var validServices = []string{kafkaSvcName}
+var validServices = []string{kafkaSvcName, registrySvcName}
```

Comment:
The registry service features should currently only be visible/accessible when the `RHOAS_DEV` flag is enabled.